packages/malabi/src/exporter/jaeger.ts

```diff
@@ -0,0 +1,15 @@
+import { JaegerExporter } from '@opentelemetry/exporter-jaeger';
+
+const JAEGER_HOST = process.env.MALABI_JAEGER_HOST || 'localhost';
+export const jaegerExporter = new JaegerExporter({
+    tags: [],
+    host: JAEGER_HOST,
+});
+
+export const getJaegerSpans = async (serviceName: string) => {
+    const JAEGER_PROTOCOL = process.env.MALABI_JAEGER_QUERY_PROTOCOL || 'http';
+    const axios = require('axios');
+    const res = await axios.get(`${JAEGER_PROTOCOL}://${JAEGER_HOST}:16686/api/traces?service=${serviceName}`)
```

Comment:
keep those, as they might be different from the exporter config. document how to use